src/main.cpp

```diff
@@ -5654,24 +5654,26 @@ bool SendMessages(CNode* pto)
         //
         // Message: getdata (non-blocks)
         //
-        while (!pto->fDisconnect && !pto->mapAskFor.empty() && (*pto->mapAskFor.begin()).first <= nNow)
+        if(!IsInitialBlockDownload())
```

Comment:
Squashed.

Well there you go, just one line of code :)

On 15/12/2015 7:38 AM, Suhas Daftuar wrote:
>
> In src/main.cpp
> <https://github.com/bitcoin/bitcoin/pull/7164#discussion_r47651822>:
>
> > @@ -5654,24 +5654,26 @@ bool SendMessages(CNode* pto)
> >          //
> >          // Message: getdata (non-blocks)
> >          //
> > -        while (!pto->fDisconnect && !pto->mapAskFor.empty() && (*pto->mapAskFor.begin()).first <= nNow)
> > +        if(!IsInitialBlockDownload())
>
> Looks good, feel free to squash
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/7164/files#r47651822>.
>